mypy/plugins/attrs.py

```diff
@@ -133,7 +134,8 @@ def argument(self, ctx: 'mypy.plugin.ClassDefContext') -> Argument:
                 # assignment, which is where you would fix the issue.
                 node = self.info[self.name].node
                 assert node is not None
-                ctx.api.msg.need_annotation_for_var(node, self.context)
+                python_version: Tuple[int, int] = sys.version_info[:2]
```

Comment:
This is slightly questionable, since the target Python version could be different from the current Python version we are running on. If we can't easily get the target Python version, I'd prefer to either not provide a Python version at all (such as giving `None` as the version -- we'd omit the hint in this case), or to provide a constant Python version for consistency (say, `(3, 7)`).